zone/command.cpp

```diff
@@ -752,6 +752,46 @@ void command_findaliases(Client *c, const Seperator *sep)
 
 void command_hotfix(Client *c, const Seperator *sep)
 {
+	auto items_count = database.GetItemsCount();
+	auto shared_items_count = database.GetSharedItemsCount();
+	if (items_count != shared_items_count) {
+		c->Message(Chat::Yellow, "Your database does not have the same item count as your shared memory.");
+
+		c->Message(
+			Chat::Yellow,
+			fmt::format(
+				"Database Count: {} Shared Memory Count: {}",
+				items_count,
+				shared_items_count
+			).c_str()
+		);
+
+		c->Message(Chat::Yellow, "You can avoid this message by creating placeholder items so you can reload them dynamically.");
```

Comment:
Maybe something like

"If you want to be able to add new items to your server while it is online, you need to create placeholder entries in the database ahead of time and do not add or remove rows / entries. Only modify the existing placeholders rows / entries to safely use #hotfix"

Reply:
👍